### ISSUE TITLE
PCA - explained variance plot.

### ISSUE THREAD

**Issue by piotrotr** [erdogant/pca#62]
Hi, 
I attempted visualized the variance explained by each principal component of my PCA:
`model_pca = pca(normalize=False, n_components=0.4) `
`results = model_pca.fit_transform(X)`
`model_pca.plot()`

Here's the output:

<img width="1237" height="870" alt="Image" src="https://github.com/user-attachments/assets/8bd79429-8359-48e2-8e86-de968e1e642f" />

Is the title correct? It says: 1 Principal Component explains [63.13%] of the variance. But it seems that this level is reached only after the 2nd Component is added. I may be misinterpreting the plot but from what I see, when there are `n` PC included, the title always gives the percentage corresponding to `n+1` PC.

**Comment by erdogant** [comment]
Thanks for mentioning this issue!

I fixed it. Update to the latest version with:
`pip install -U pca
`

let me know if you notice anything else that is off.
